### DISCUSSION TITLE
JCenter Sunset

### DISCUSSION THREAD

**Discussion by GabyZu** [openTCS/opentcs#161] [Q&A]
Is it possible to make available openTCS version 4.18 on Maven given that JCenter's sunset on 8/15/2024?

**Comment by swltr** [comment]
JCenter's fate is unfortunate, and it would certainly be nice to have these artifacts in Maven Central, but at least currently, we do not intend to spend resources on migrating old releases. (If you really need to keep a dependency on openTCS 4.18, my suggestion is to build it yourself and place the artifacts in a local repository.)
